### DISCUSSION TITLE
Wails funcs in other packages

### DISCUSSION THREAD

**Discussion by illud** [wailsapp/wails#2662] [General]
Hello, 

Is there a way to put functions in other packages let's say controllers and then make App context recognize those functions?
i want to do this because i have a lot of functions so having that in one place is hard to maintain.

Thanks.

**Comment by misitebao** [comment]
No problem at all, just like developing any other Go application. Wails doesn't limit how you develop.

**Reply by illud** [reply]
Hello, but how?
